frontend/src/components/NegotiationSidebar.vue

```diff
@@ -70,6 +70,11 @@
       </li>
       <li class="list-group-item p-2 btn-sm border-bottom-0">
         <PDFButton class="mt-2" :negotiation-pdf-data="negotiation" />
+        <MergedPDFButton
+          class="mt-2"
+          :negotiation-pdf-data="negotiation"
+          data-cy="merged-pdf-button"
+        />
```

Comment:
please add a loading spinner ( for larger pdf this can take more than 10 seconds)